charon-ml/src/LlbcAstUtils.ml

```diff
@@ -1,6 +1,18 @@
 include GAstUtils
 open LlbcAst
 open Utils
+open Collections
+
+let fun_decl_list_from_crate (crate : crate) : fun_decl list =
+  snd (List.split (FunDeclId.Map.bindings crate.fun_decls))
+
+let get_fun_args (fun_decl : fun_decl) : var list =
+  match fun_decl.body with
+  | Some body ->
+      let input_number = body.arg_count in
+      let input_list = snd (List.split_at body.locals 1) in
+      fst (List.split_at input_list input_number)
+  | None -> []
```

Comment:
This is misleading: a function without a body is not the same thing as a function that takes no arguments; this could cause issues. Please either return `var list option` or operate on a `fun_body`.